src/libraries/System.Text.Json/src/System/Text/Json/Serialization/JsonConverter.cs

```diff
@@ -71,6 +75,12 @@ internal bool ShouldFlush(Utf8JsonWriter writer, ref WriteStack state)
         /// </summary>
         internal abstract bool WriteCoreAsObject(Utf8JsonWriter writer, object? value, JsonSerializerOptions options, ref WriteStack state);
 
+        /// <summary>
+        /// Loosely-typed WriteWithQuotes() that forwards to strongly-typed WriteWithQuotes().
+        /// </summary>
+        internal virtual void WriteWithQuotesAsObject(Utf8JsonWriter writer, object value, JsonSerializerOptions options, ref WriteStack state)
```

Comment:
The existing pattern is to make these abstract. Here and the changes in JsonConverter<T>.